### PR TITLE
Don't clone git-open recursively

### PR DESCRIPTION
**Pull request by davidosomething**
git-open includes its test framework, don't need it

### CODE REVIEW

**Comment by psprint** [comment]
Isn't it that disabling the recursive clone will take away the "included test framework" from `git-open`? This sounds like some negation of the argument and the action.

**Comment by davidosomething** [comment]
can you rephrase that?
right now when you include git-open as program, the repo is cloned with `--recursive` since that's the default cloneopt
with --recursive, you also end up cloning the `bash assertion testing system`, which is not useful since you want `as'program'`

the only way to unset cloneopt is to set it to something, in this case an empty space

**Comment by psprint** [comment]
Ah I see what you meant. Ok, I'll merge.

**Comment by psprint** [comment]
I've updated Zinit so that passing empty `cloneopts` also disables the default options. I've updated the README too.
